### PR TITLE
ACS-6661 Process new content during updates events.

### PR DESCRIPTION
**Pull request by tpage-alfresco**

### CODE REVIEW

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/in/EventProcessor.java [hunk 1]
```diff
@@ -76,19 +79,58 @@
...
+            return latestContentPresent;
+        }
+        else if (isEventTypeUpdated(event))
+        {
+            Optional<ContentInfo> oldContentInfo = ofNullable(event.getData().getResourceBefore()).map(NodeResource::getContent);
+            // If content isn't mentioned in the resourceBefore then there was no change to the content.
+            if (oldContentInfo.isEmpty())
+            {
+                return false;
+            }
```
[the remark this answers is not in the document]
Comment (tpage-alfresco) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/in/EventProcessor.java:128]: I thought the logic here was complex enough to warrant detailed explanations in comments and so didn't try to condense it. What do you guys think about this?

[the remark this answers is not in the document]
Comment (Pawel-608) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/in/EventProcessor.java:128]: I would leave it as it is as this logic is really complex

[the remark this answers is not in the document]
Comment (krdabrowski) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/in/EventProcessor.java:128]: The logic is not obvious, so some descriptions are very useful, but I think you can condense this particular condition.

**Review by krdabrowski — APPROVED** [review]
LGTM

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/in/EventProcessor.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (tpage-alfresco) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/in/EventProcessor.java]: I've come up with `return oldContentInfo.isPresent() && latestContentPresent;` which seems ok too. Let me know what you think.

**Pull request merged**